charger/easee.go

```diff
@@ -338,8 +322,8 @@ func (c *Easee) Enable(enable bool) error {
 		resp, err := c.Post(uri, request.JSONContent, request.MarshalJSON(data))
 		if err == nil {
 			resp.Body.Close()
-			return err
 		}
+		return err
```

Comment:
@naltatis das `resume_charging` braucht es in diesem Fall nicht mehr?

Reply:
Doch, das resume muss gesendet. Kann ja sein, dass vorher Pause war.

Reply:
Ok, hab ich geändert. Das war vorher allerdings nicht so!